tools/DependencyListGenerator/GlobalSuppressions.cs

```diff
@@ -18,3 +18,4 @@
 
 [assembly: SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1600:Elements should be documented", Justification = "Reviewed.")]
 [assembly: SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1604:Element documentation should have summary", Justification = "Reviewed.")]
+[assembly: SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1633:File should have header", Justification = "Preserving original license.")]
```

Comment:
I really would like to scope it only for a namespace or a folder, but somehow stylecop does not address it.